src/AutoVizuA11y.tsx

```diff
@@ -314,7 +358,9 @@ const AutoVizuA11y = ({
 				style={{ textIndent: "-10000px" }}
 				className="a11y_desc visually-hidden"
 				data-testid="a11y_desc"
-				onFocus={handleFocus}
+				onFocus={() => {
```

Comment:
Same thing here. Can't we pass the function reference here and actually get the value of `alertDiv` inside of it?

Reply:
Same point as before :D